dragonfly/building.py

```diff
@@ -183,26 +321,60 @@ def shade_representation(self, tolerance):
     def auto_assign_top_bottom_floors(self):
         """Set the first Story as the ground floor and the last to be the top.
 
-        If the last floor in the unique_stories has a multiplier greater than 1, this
-        method will automatically create a new unique story at the top of the building
-        with the is_top_floor property set to True.
+        Note that this methods does not change the number of unique_stories.
 
-        Note that this method may not give the desired result if the building has
+        Also note that this method may not give the desired result if the building has
         several top floors (like towers of different heights connected by a plinth).
         For such a case, it might be better to manually assign the is_top_floor property
         of each of the relevant Stories.
         """
         self._unique_stories[0].is_ground_floor = True
-        if self._unique_stories[-1].multiplier == 1:
-            self._unique_stories[-1].is_top_floor = True
-        else:
-            top = self._unique_stories[-1].duplicate()
-            move_vec = Vector3D(0, 0, top.floor_to_floor_height * top.multiplier)
-            top.move(move_vec)
-            top.is_top_floor = True
-            top.multiplier = 1
-            self._unique_stories[-1].multiplier = self._unique_stories[-1].multiplier - 1
-            self._unique_stories = self._unique_stories + (top,)
+        self._unique_stories[-1].is_top_floor = True
+
+    def separate_top_bottom_floors(self):
```

Comment:
I'm using it in my tests right now as a shortcut to save me from writing 2 lines instead of 1. I think it's a useful method whenever someone is creating a building story-by-story and not using the classmethods.